UI/Web/src/app/book-reader/_components/_drawers/view-edit-annotation-drawer/view-edit-annotation-drawer.component.ts

```diff
@@ -367,4 +371,28 @@ export class ViewEditAnnotationDrawerComponent implements OnInit {
       this.offcanvasService.dismiss();
     });
   }
+
+  handleLikeChange() {
+    const annotation = this.annotation();
+    if (!annotation) return;
+
+    if (annotation.ownerUserId === this.accountService.currentUserSignal()!.id) return;
```

Comment:
We can cache the userId with ?.id so we can early return and avoid the null suppression afterwards